src/Generator/ValueGenerator.php

```diff
@@ -143,7 +149,7 @@ public function deleteConstant($constant)
     /**
      * Return constant list
      *
-     * @return ArrayObject
+     * @return SplArrayObject|StdlibArrayObject
```

Comment:
Return a different thing of Stdlib version is a BC Break.

Both objects share the same methods and signature so nobody should find the difference except if typehint the expected type.